src/components/Layout/Header/NavBar/Native/NativeMenu.tsx

```diff
@@ -1,42 +1,27 @@
-import { ChevronRightIcon, EditIcon } from '@chakra-ui/icons'
+import { ChevronRightIcon } from '@chakra-ui/icons'
 import { Button, MenuDivider, MenuItem } from '@chakra-ui/react'
 import React, { useCallback } from 'react'
-import { useTranslate } from 'react-polyglot'
+import { AccountsManagementMenuItem } from 'components/Layout/Header/NavBar/AccountsManagementMenuItem'
```

Comment:
I believe that's a great change and component abstraction, a bunch of repeated code over various wallet menus for the account management feature, however, if we're refactoring this, we should also do so in `<DemoMenu />`, `<KeepKeyMenu />` and `<LedgerMenu />`, though that's outside of the scope of this PR (and so is native diff for a MM feature).

Let's keep it a separate PR and tackle holistically? 🙏🏽

Reply:
Alright, will push a PR tomorrow in the morning for the refactoring